crates/ruma-client-api/src/sync/sync_events/v4.rs

```diff
@@ -627,8 +627,7 @@ pub struct AccountData {
 
 /// Receipt extension configuration.
 ///
-/// Not yet part of the spec proposal. Taken from the reference implementation
-/// <https://github.com/matrix-org/sliding-sync/blob/main/sync3/extensions/receipts.go>
+/// According to [MSC3960](https://github.com/matrix-org/matrix-spec-proposals/pull/3960)
 #[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq)]
 #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
 pub struct ReceiptConfig {
```

Comment:
Can you rename this type to `ReceiptsConfig` so it's consistent with the field name?